src/lib/route-path.ts

```diff
@@ -20,4 +23,23 @@ export class RoutePath {
   static readonly CONTACT_US = "/contact-us";
   static readonly PRIVACY_POLICY = "/privacy-policy";
   static readonly TERMS_AND_CONDITIONS = "/terms-and-conditions";
+
+  static readonly ARTICLES = "/articles";
+  static readonly ABOUT_US = "/about";
+  static readonly BE_A_DONOR = "/be-a-donor";
+  static readonly INFO = "/info";
+  static readonly FAQ = "/faq";
+
+  static readonly ACCOUNT_DASHBOARD = (accountType: Url) => {
+    if (accountType === "ADMIN") {
+      return RoutePath.ADMIN_DASHBOARD_PAGE;
+    }
+    if (accountType === "VET") {
+      return RoutePath.VET_DASHBOARD_PAGE;
+    }
+    if (accountType === "USER") {
+      return RoutePath.USER_DASHBOARD_PAGE;
+    }
+    return "/";
```

Comment:
Decided on a fallback instead of throwing an error since an error would break the page once logged out or for new comers who are not any of the account types